elated-egrets/bangazon-internal-site-2.0
# J redit user

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout JRedit_user
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. register a new user
1. click the register button
1. once at the home page go to view the profile
1. make sure necessary info is displayed 
1. click the update button
1. change the info
1. update the changes
1. make sure info is the same as what was updated

## What you changed
user profile and relationship. added additional fields for address
## What was added to read me
N/A
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Have 2 teammates approved the PR?

Then Ship It!
